pojo-generator-core/src/main/java/uk/gov/justice/generation/pojo/generators/plugin/typename/SuportJavaOptionalsPlugin.java

```diff
@@ -10,7 +10,7 @@
 import com.squareup.javapoet.ParameterizedTypeName;
 import com.squareup.javapoet.TypeName;
 
-public class OptionalTypeNamePlugin implements TypeNamePlugin {
+public class SuportJavaOptionalsPlugin implements TypeModifyingPlugin {
```

Comment:
Just a spelling mistake...two 'p's in Support above

Reply:
Ooops